app/templates/gulp/_env-config.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var gulp = require('gulp');
+var fs = require('fs');
+var argv = require('yargs').argv;
+var $ = require('gulp-load-plugins')();
```

Comment:
To allow reading from config add

   var conf = require('./conf');
   var path = require('path');